### PR TITLE
MMB: Trip overview add icons, model and terminal

### PR DESCRIPTION
**Pull request by tbergq**

### CODE REVIEW

File: app/MMB/src/scenes/tripOverview/TimelineDeparture.js [hunk 1]
```diff
@@ -31,24 +38,44 @@ function TimelineDeparture(props: Props) {
...
+                  id="mmb.flight_overview.timeline.terminal"
+                  values={{
+                    terminal,
+                    city,
+                    iata,
+                  }}
+                />
+              }
+            />
+          )}
```
Comment (RobinCsl) [inline app/MMB/src/scenes/tripOverview/TimelineDeparture.js]: There should also be one like that for the TimelineArrival_routeStop where the icon is the landing plane.

File: packages/localization/src/vocabularies/ManageMyBooking.js [hunk 2]
```diff
@@ -70,6 +70,8 @@ export default {
     "Warning: you will depart from a different place. You'll arrive at __1_arrival__ and leave from __2_departure__.",
   'mmb.flight_overview.timeline.warning.different_airport_return_first':
     'Warning: your departure point differs from your return one. You will depart from __1_departure__ and return to __2_arrival__.',
+  'mmb.flight_overview.timeline.terminal':
+    '__1_city__ - __2_iata__ - Terminal - __3_terminal__',
```
Comment (RobinCsl) [inline packages/localization/src/vocabularies/ManageMyBooking.js]: I think it should be 
```
'__1_city__ __2_iata__ - Terminal - __3_terminal__'
```

File: app/MMB/src/scenes/tripOverview/TimelineDeparture.js (hunk 1 above)
Comment (tbergq) [reply app/MMB/src/scenes/tripOverview/TimelineDeparture.js]: It is no landing icon in the current app

Comment (tbergq) [reply app/MMB/src/scenes/tripOverview/TimelineDeparture.js]: I will do this in another PR. There is still more missing parts here anyways.

**Pull request merged**